README.md

```diff
@@ -1,5 +1,15 @@
 # Crowdstaking
 
+### Terminology
```

Comment:
Discussed this with Ron in Discord, that this terminology is all over the codebase but never explicitly defined? Can exclude if need be.